### ISSUE TITLE
showcase, registry, gallery

### ISSUE THREAD

**Issue by timelyportfolio** [ramnathv/htmlwidgets#63]
While the current `htmlwidgets` [showcase](http://www.htmlwidgets.org/showcase_leaflet.html), I'm not sure it will be the best way to represent what I [believe](http://buildingwidgets.org) and hope to be an explosion of `htmlwidgets`.  Possible related to #55, but I thought a couple great examples would be 

1. [d3.js gallery by @d3visualization](http://christopheviau.com/d3list/gallery.html) - [Github repo](https://github.com/biovisualize/d3visualization)
1. [d3.js site](http://d3js.org/)
2. [biojs registry](http://registry.biojs.net/#/) - [Github repo](https://github.com/biojs/registry-ui)

**Comment by jjallaire** [comment]
Yes, that Showcase page definitely won't scale and is intended more as a
motivator to widget developers than as a comprehensive resource for end
users. Some type of lightweight gallery where we can list one and all would
be awesome.

**Comment by ramnathv** [comment]
I will push one which can be updated via a Google Spreadsheet. We can provide developers with a Google Form to populate the spreadsheet.

**Comment by smartinsightsfromdata** [comment]
@ramnathv any news on the Google Spreadsheet?

Thanks!

**Comment by ramnathv** [comment]
I should have it ready by end-of-week.

**Comment by jpmarindiaz** [comment]
Here is my attempt: 
http://jpmarindiaz.com/static/htmlwidgets-gallery/

It uses another htmlwidget I built (https://github.com/jpmarindiaz/isotope). It is still in very early stages but it works.

Feel free to contribute more examples and fill/correct this google spreadsheet. 
https://docs.google.com/spreadsheets/d/1Ay95DIlVL4Yn5bsByqPq-VjKcjjlocRpsCg7tGb32VA/edit#gid=0

If you find it valuable it would be cool to include it in the htmlwidgets.org page

Cheers

**Comment by hafen** [comment]
@jpmarindiaz this looks like a great collection.  I can't seem to edit the spreadsheet or upload to dropbox.  But it would be nice to have rbokeh added to the list.  

https://github.com/bokeh/rbokeh
http://hafen.github.io/rbokeh/

**Comment by hafen** [comment]
I'd also like to propose another mechanism to consider to deploy/organize this.  Rather than a google spreadsheet + translation to html, one idea would be to host the page on github pages and use the jekyll yaml file to fill out a template of widget entries.  To make it collaborative yet controlled we can handle new entries or edits through pull requests.  Then the page would be automatically re-generated when new content is added and approved.  That's one idea.

**Comment by jcheng5** [comment]
@jpmarindiaz Nice list!

FYI, I get a ton of error 503 from dropbox URLs when I try to visit http://jpmarindiaz.com/static/htmlwidgets-gallery/, and all the images and CSS are broken.

**Comment by jpmarindiaz** [comment]
@hafen feel free to edit the spreadsheet, you should be able to edit it now.

Great idea I actually built the html with the isotope htmlwidget. Maybe we han have the gallery in the github pages of this package.

Here you can find the original data I collected. I figured I would go with the google doc to gather some initial data and then handle everything on the isotope package as an example.
https://github.com/jpmarindiaz/isotope/blob/master/inst/data/htmlwidgets.yaml

**Comment by jpmarindiaz** [comment]
@jcheng5 yes, there is something wrong with dropbox right now. I will move the images.

**Comment by ramnathv** [comment]
Great work! I would agree with @hafen that a github PR based mechanism would work better so that the page is auto-generated.

**Comment by hafen** [comment]
@jpmarindiaz I should have looked at how you are doing it - looks like you're already on the way to what I was suggesting!  If it's useful I can make a quick mock up merging what you've done with jekyll.  

If github is the way to go, @ramnathv, any suggestion on where repo with the gh-pages should be?  I suppose it can be linked with any URL so it's probably not a big deal, but who has permissions to merge PRs, etc. is a consideration.

**Comment by jpmarindiaz** [comment]
How about this? @ramnathv @hafen @jcheng5 
http://jpmarindiaz.github.io/isotope/htmlwidgets-gallery/

Just go ahead PR the gh-branch in https://github.com/jpmarindiaz/isotope
There you can find the csv file with the list of the htmlwidgets I have spotted. 
(or shall we use a yaml??? it is also there)

Just edit the csv file and re-generate the html as in:
https://github.com/jpmarindiaz/isotope/blob/gh-pages/htmlwidgets-gallery/htmlwidgets-gallery.Rmd

**Comment by jcheng5** [comment]
Just FYI (and sorry to be "that" guy) but the alignment is all messed up on
my phone (iOS).


-- 
Sent from my phone
